blake3/blake3.go

```diff
@@ -0,0 +1,44 @@
+// Copyright 2021 OCI Contributors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package blake3
+
+import (
+	"hash"
+
+	"github.com/opencontainers/go-digest"
+	"github.com/zeebo/blake3"
```

Comment:
Is there any chance that this implementation will substantively change if there is a canonical (e.g. `golang.org/x/crypto/...`) package for blake3?

Reply:
No. The cryptoHash interface is a subset of the hash interface in standard Go.

Reply:
Oh, I meant the hash algorithm itself, not the interface itself.:-)

Reply:
The algorithm itself has been released and specified. It's in production use in a bunch of places. I'm pretty sure it's considered stable at this point.

Reply:
👍